From d3352ffb5bac201ce3e88a87b9f02a695a723c02 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Mon, 6 Aug 2012 15:44:56 +0200
Subject: [PATCH 51/53] remove json, bodhi and ureport

- json is not in rhel and both ureport and bodhi depends on it
---
 configure.ac            |  2 --
 doc/Makefile.am         |  1 -
 po/POTFILES.in          |  3 ---
 src/lib/Makefile.am     |  2 +-
 src/plugins/Makefile.am | 40 +++-------------------------------------
 5 files changed, 4 insertions(+), 44 deletions(-)

diff --git a/configure.ac b/configure.ac
index 459db9c..fef7443 100644
--- a/configure.ac
+++ b/configure.ac
@@ -74,8 +74,6 @@ PKG_CHECK_MODULES([XMLRPC], [xmlrpc])
 PKG_CHECK_MODULES([XMLRPC_CLIENT], [xmlrpc_client])
 PKG_CHECK_MODULES([CURL], [libcurl])
 PKG_CHECK_MODULES([GNOME_KEYRING], [gnome-keyring-1])
-PKG_CHECK_MODULES([JSON_C], [json])
-PKG_CHECK_MODULES([RPM], [rpm])
 PKG_CHECK_MODULES([BTPARSER], [btparser])
 
 # Just PKG_CHECK_MODULES([PYTHON], [python]) works only with python2.7+
diff --git a/doc/Makefile.am b/doc/Makefile.am
index 70e0fb0..542fe4c 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -16,7 +16,6 @@ MAN1_TXT += reporter-print.txt
 MAN1_TXT += reporter-rhtsupport.txt
 MAN1_TXT += reporter-upload.txt
 MAN1_TXT += report.txt
-MAN1_TXT += abrt-bodhi.txt
 
 MAN5_TXT =
 MAN5_TXT += report_event.conf.txt
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 9fac721..91e1415 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -23,7 +23,6 @@ src/plugins/report_Logger.xml.in
 src/plugins/report_Mailx.xml.in
 src/plugins/report_RHTSupport.xml.in
 src/plugins/report_Uploader.xml.in
-src/plugins/report_uReport.xml.in
 src/plugins/reporter-bugzilla.c
 src/plugins/reporter-kerneloops.c
 src/plugins/reporter-mailx.c
@@ -32,6 +31,4 @@ src/plugins/reporter-rhtsupport.c
 src/plugins/reporter-rhtsupport-parse.c
 src/plugins/reporter-upload.c
 src/plugins/rhbz.c
-src/plugins/abrt-bodhi.c
 src/report-newt/report-newt.c
-src/plugins/ureport.c
diff --git a/src/lib/Makefile.am b/src/lib/Makefile.am
index 33e7f22..2d08fee 100644
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -95,7 +95,7 @@ libabrt_web_la_SOURCES = \
     abrt_curl.h abrt_curl.c \
     abrt_xmlrpc.h abrt_xmlrpc.c \
     proxies.h proxies.c \
-    ureport.h json.c
+    ureport.h
 libabrt_web_la_CPPFLAGS = \
     -Wall -Wwrite-strings -Werror \
     -I$(srcdir)/../include \
diff --git a/src/plugins/Makefile.am b/src/plugins/Makefile.am
index f0d541f..0284b3c 100644
--- a/src/plugins/Makefile.am
+++ b/src/plugins/Makefile.am
@@ -7,9 +7,7 @@ bin_PROGRAMS = \
     reporter-upload \
     reporter-mailx \
     reporter-print \
-    report \
-    abrt-bodhi \
-    reporter-ureport
+    report
 
 pluginsconfdir = $(PLUGINS_CONF_DIR)
 
@@ -30,8 +28,7 @@ dist_events_DATA = \
     report_Mailx.xml \
     report_RHTSupport.xml \
     report_Kerneloops.xml \
-    report_Uploader.xml \
-    report_uReport.xml
+    report_Uploader.xml
 
 @INTLTOOL_XML_RULE@
 
@@ -52,8 +49,7 @@ EXTRA_DIST = \
     report_Mailx.xml.in \
     report_RHTSupport.xml.in \
     report_Kerneloops.xml.in \
-    report_Uploader.xml.in \
-    report_uReport.xml.in
+    report_Uploader.xml.in
 
 $(DESTDIR)/$(DEBUG_INFO_DIR):
 	$(mkdir_p) '$@'
@@ -204,34 +200,4 @@ report_CPPFLAGS = \
 report_LDADD = \
     ../lib/libreport.la
 
-abrt_bodhi_SOURCES = \
-    abrt-bodhi.c
-abrt_bodhi_CPPFLAGS = \
-    -I$(srcdir)/../include \
-    -I$(srcdir)/../lib \
-    $(GLIB_CFLAGS) \
-    $(LIBREPORT_CFLAGS) \
-    $(JSON_C_CFLAGS) \
-    $(RPM_CFLAGS) \
-    -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror
-abrt_bodhi_LDADD = \
-    $(JSON_C_LIBS) \
-    $(RPM_LIBS) \
-    ../lib/libreport.la \
-    ../lib/libabrt_web.la
-
-reporter_ureport_SOURCES = \
-    ureport.c
-reporter_ureport_CPPFLAGS = \
-    -I$(srcdir)/../include \
-    -I$(srcdir)/../lib \
-    $(GLIB_CFLAGS) \
-    $(LIBREPORT_CFLAGS) \
-    -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror
-reporter_ureport_LDADD = \
-    ../lib/libreport.la \
-    ../lib/libabrt_web.la
-
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
-- 
1.7.11.2

